DuckDuckGo/NewTabPageManager.swift

```diff
@@ -35,18 +35,23 @@ final class NewTabPageManager: NewTabPageManaging, NewTabPageDebugging {
 
     var appDefaults: AppDebugSettings
     let featureFlagger: FeatureFlagger
+    let internalUserDecider: InternalUserDecider
 
     init(appDefaults: AppDebugSettings = AppDependencyProvider.shared.appSettings,
-         featureFlager: FeatureFlagger = AppDependencyProvider.shared.featureFlagger) {
-        
+         featureFlager: FeatureFlagger = AppDependencyProvider.shared.featureFlagger,
+         internalUserDecider: InternalUserDecider = AppDependencyProvider.shared.internalUserDecider) {
+
         self.appDefaults = appDefaults
         self.featureFlagger = featureFlager
+        self.internalUserDecider = internalUserDecider
     }
 
     // MARK: - HomeTabManaging
 
     var isNewTabPageSectionsEnabled: Bool {
-        isLocalFlagEnabled && isFeatureFlagEnabled
+        let isLocalFlagInEffect = isLocalFlagEnabled && internalUserDecider.isInternalUser
+        
+        return isLocalFlagInEffect || isFeatureFlagEnabled
```

Comment:
Remote flag is now `.remoteDevelopment`, so either local flag needs to be set or enabled in privacy config, both only for internal users.